src/main/java/hudson/plugins/copyartifact/CopyArtifact.java

```diff
@@ -543,7 +547,7 @@ public void perform(@NonNull Run<?, ?> build, @NonNull FilePath workspace, @NonN
             if (!ok) {
                 throw new AbortException(Messages.CopyArtifact_FailedToCopy(expandedProject, expandedFilter));
             }
-        } else if (src instanceof MatrixBuild) {
+        } else if (jenkins.getPlugin("matrix-project") != null && src instanceof MatrixBuild) {
```

Comment:
Compare #14 (amending 4fd5c3efb7bee0df3c93204813340ee3fc8e7373). Without this, the new test fails:

```
Started
[Pipeline] Start of Pipeline
[Pipeline] node
Running on Jenkins in …/copyartifact-plugin/target/tmp/j h11474560698679713783/workspace/downstream
[Pipeline] {
[Pipeline] copyArtifacts
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
java.lang.ClassNotFoundException: hudson.matrix.MatrixBuild
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
	at jenkins.util.URLClassLoader2.findClass(URLClassLoader2.java:35)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:592)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
Also:   org.jenkinsci.plugins.workflow.actions.ErrorAction$ErrorId: af406f0c-e047-466c-874b-a56e04bfb18a
Caused: java.lang.NoClassDefFoundError: hudson/matrix/MatrixBuild
	at hudson.plugins.copyartifact.CopyArtifact.perform(CopyArtifact.java:546)
	at jenkins.tasks.SimpleBuildStep.perform(SimpleBuildStep.java:123)
	at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:101)
	at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:71)
	at org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Finished: FAILURE
```